crates/meroctl/src/cli/context/ls.rs

```diff
@@ -0,0 +1,57 @@
+use calimero_primitives::identity::Context;
+use clap::Parser;
+use eyre::eyre;
+use reqwest::Client;
+use serde::{Deserialize, Serialize};
+
+use super::get_ip;
+use crate::cli::RootArgs;
+use crate::config::ConfigFile;
+#[derive(Debug, Serialize, Deserialize)]
+pub struct GetContextsResponse {
+    data: Vec<Context>,
+}
+
+#[derive(Debug, Parser)]
+pub struct LsCommand {}
+
+impl LsCommand {
+    pub async fn run(self, root_args: RootArgs) -> eyre::Result<()> {
+        let path = root_args.home.join(&root_args.node_name);
+        if ConfigFile::exists(&path) {
+            if let Ok(config) = ConfigFile::load(&path) {
+                let multiaddr = config
+                    .network
+                    .server
+                    .listen
+                    .first()
+                    .ok_or_else(|| eyre!("No address."))?;
+                let base_url = get_ip(multiaddr)?;
+                let url = format!("{}admin-api/contexts-dev", base_url);
+
+                let client = Client::new();
+                let response = client.get(&url).send().await?;
+
+                if response.status().is_success() {
+                    let api_response: GetContextsResponse = response.json().await?;
+                    let contexts = api_response.data;
+
+                    println!("Contexts:");
+                    for context in contexts {
+                        println!("App ID: {}", context.application_id);
+                        println!("Context ID: {}", context.id);
+                        println!();
```

Comment:
why's there an extra new line at the end? personal preference?

Reply:
Just a personal preference, I think it is easier to read when there are a lot of contexts.
Probably should also return a `json` so its usable if piped into other cli programs.

Reply:
ah, okay..

how about listing just the context IDs by default

```console
$ meroctl context ls
94G5Y6mNvHAZTS61JNCF1rvj4kwyn6m7Ns3TtexztFjW
3vthqNdhcaFHjRCdGcirjvX5qFcUjGNYaUCXs3oyBPQp
3NTfcKz5j4iDLi4mUiqsgpLdgdWtomD7Jd1jZTCV8MjH
```

and something like this with `-l, --long`:

```console
$ meroctl context ls -l
├─[94G5Y6mNvHAZTS61JNCF1rvj4kwyn6m7Ns3TtexztFjW]
│   ├─[Application ID]: Hcf7pWQqFNzCRvkyHSdjDQjagKh4QKhoZYKBfuANeqgt
│   │   ├─[Name]: only-peers
│   │   └─[Version]: 0.2.0
│   └─[Identities]: 12
│       ├─ 7e7d5Rag4HRx2Tp2BTMLqi4m8nn2bswHFRzjEEDLrjs8
│       ├─ ...
│       ├─ ...
│       └─ 6w931mWdGyfX3XC5QjP3qhNLE6SC2ybYGHbV5L8vLGwn
├─[3vthqNdhcaFHjRCdGcirjvX5qFcUjGNYaUCXs3oyBPQp]
│   ├─[Application ID]: D36d8HH1gwEtc7furQdiXr8GvGysCKYR6JevcQgN3HeH
│   │   ├─[Name]: rock-paper-scissors
│   │   └─[Version]: 0.1.0
│   └─[Identities]: 3
│       ├─ 3imfDqhqvFNrsn3dmgK9315vyuvtLWu22vY7Pd8vwZfp
│       ├─ GNu4AVzgDU9QRKScg1d2qMM3XL7FmBRatkjPhQWDPK9B
│       └─ 7vkuKGGYapXnkT4dL5Sapt5MLKpjzCaJ3S5HehQFrA1M
└─[3NTfcKz5j4iDLi4mUiqsgpLdgdWtomD7Jd1jZTCV8MjH]
    ├─[Application ID]: 3D9R6ZsPL6j8JawaAZo9Jwi8uGJGHt6NXSqA16oP8cqR
    │   ├─[Name]: kv-store
    │   └─[Version]: 0.1.0
    └─[Identities]: 2
        ├─ EfHYJ3oNhg5QBQgGe8ZxSLtS5Tpu2tBVnH9HfAX2JwgU
        └─ CJDuNj6RaQQFydu8ueWTXR1VsZZtdoFmtJp7YaScS6vd
```